src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -225,10 +231,11 @@ public boolean isBuildPlanEnabled(final String planId) {
     /**
      * Delete the build plan with given identifier from Bamboo.
      *
+     * @param projectKey The key of the project of the related programming exercise
```

Comment:
consistently remove JavaDoc from subclasses if the documentation is already completely understandable in the interface

Reply:
I removed the JavaDoc in this case, we now only have the doc in the interface